networking/assigning-egress-ips.adoc

```diff
@@ -0,0 +1,50 @@
+[id="assigning-egress-ips"]
+= Assigning egress IPs to a project
+include::modules/common-attributes.adoc[]
+:context: egress-ips
+
+toc::[]
+
+As a cluster administrator you can configure OpenShift Software Defined Network
+(SDN) to assign one or more egress IP addresses to a project. All outgoing
+external connections from the specified project will share the same, fixed
+source IP, meaning that any external resources can recognize the traffic. An
+egress IP address assigned to a project is different from the egress router,
+which is used to send traffic to specific destinations.
+
+You can assign egress IP addresses to namespaces by setting the `egressIPs`
+field of the `NetNamespace` resource. Once an egress IP is associated with a
+project, OpenShift SDN allows you to assign egress IPs to hosts in two ways:
+
+* In the _automatically-assigned_ approach, an egress IP address range is
+assigned to a node. This way is the preferred approach.
+* In the _manually assigned_ approach, a list of one or more egress IP address
+is assigned to a node.
+
+With automatically-assigned egress IPs, you set the `egressCIDRs` field of
```

Comment:
Per IBM Style Guide, "Do not use a hyphen after an adverb or adjective that ends in -ly." So, manually assigned egress IP and automatically-assigned egress IPs would be correct.